lib/reek/cli/command/todo_list_command.rb

```diff
@@ -36,13 +36,13 @@ def scan_for_smells
         end
 
         def groups_for(smells)
-          @groups ||= begin
-            Hash[
-              smells.group_by(&:smell_type).map do |smell_type, smells_for_type|
-                [smell_type, { 'exclude' => smells_for_type.map(&:context) }]
+          @groups ||=
+            begin
+              todos = smells.group_by(&:smell_class).map do |smell_class, smells_for_class|
```

Comment:
> the can of worms above

We once tried to get conversion to YAML working for SmellWarning, using `#encode_with` for Psych. We removed it in the end: fbda199884b834e74e1899daf32d5ee0cfcb5788

> Use the string

I'd like to do this, but for a good implementation, I think it needs a bit of refactoring. So, how about deferring that to another PR?